backend/siarnaq/settings.py

```diff
@@ -411,8 +411,8 @@ class Production(Base):
 
     GCLOUD_BUCKET_PUBLIC = "mitbattlecode-production-public"
     GCLOUD_BUCKET_SECURE = "mitbattlecode-production-secure"
-    GCLOUD_TOPIC_COMPILE = "production-siarnaq-compile"
-    GCLOUD_TOPIC_EXECUTE = "production-siarnaq-execute"
+    GCLOUD_TOPIC_COMPILE = "projects/mitbattlecode/topics/production-siarnaq-compile"
+    GCLOUD_TOPIC_EXECUTE = "projects/mitbattlecode/topics/production-siarnaq-execute"
```

Comment:
Is ok to leave like this - although ideal solution would be to use [this method](https://cloud.google.com/python/docs/reference/pubsub/latest/google.cloud.pubsub_v1.publisher.client.Client#google_cloud_pubsub_v1_publisher_client_Client_topic_path) instead in `backend/siarnaq/api/compete/managers.py:43`. I can do this separately if you'd prefer (just make an issue and assign it to me).

Would look something like reverting these changes and then writing
```python
topic=publish_client.topic_path(settings.GCLOUD_PROJECT, self._publish_topic),
```

Reply:
#417 (cuz prioritizng)